packages/teams-js/test/private/files.spec.ts

```diff
@@ -548,54 +550,123 @@ describe('files', () => {
   });
 
   describe('openDownloadFolder', () => {
-    it('should not allow calls before initialization', () => {
-      expect(() => files.openDownloadFolder(null, emptyCallback)).toThrowError(
-        'The library has not yet been initialized',
-      );
-    });
+    describe('v1', () => {
+      it('should not allow calls before initialization', () => {
+        expect(() => files.openDownloadFolder(null, emptyCallback)).toThrowError(
+          'The library has not yet been initialized',
+        );
+      });
+      Object.keys(FrameContexts)
+        .map(k => FrameContexts[k])
+        .forEach(frameContext => {
+          it(`should not allow calls from ${frameContext} context`, async () => {
+            if (frameContext === FrameContexts.content) {
+              return;
```

Comment:
Unit Test case guideline states we avoid the use of return, reference code : Object.keys(FrameContexts).forEach(k => { const context = FrameContexts[k]; if (allowedContexts.some(allowedContext => allowedContext === context)) {
add test cases in if block.

Reply:
are you sure, i've seen that return case in other modules, calendar and mail being examples. I don't see this in the unit test case guidelines anywhere but I may be missing it.  I'm fine changing it, but I'd like to know where the guideline is so that I can avoid these in the future, i've been reading off of the guide @erinha  posted

Reply:
Check the note where the preference is using `async/await` 
https://office.visualstudio.com/MetaOS/_wiki/wikis/MetaOS.wiki/43604/Unit-Test-Guidelines